Samples.Swift/Samples.Swift/ViewControllers/CheckoutViewController/CheckoutViewController.swift

```diff
@@ -0,0 +1,444 @@
+//
+//  CheckoutViewController.swift
+//  paypalcheckout-iOS-swift-sample
+//
+//  Created by Jax DesMarais-Leder on 3/10/21.
+// Copyright © 2021 PayPal. All rights reserved.
+//
+
+import UIKit
+import PayPalCheckout
+
+class CheckoutViewController: UIViewController, AddItemViewControllerDelegate {
+
+  let tableView = UITableView()
+  let checkoutButton = UIButton()
+  var checkoutFlowOption = UISegmentedControl()
+  var items = [PurchaseUnit.Item]()
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    setupSampleItem()
+    setupUI()
+    setupTableView()
+    setupConstraints()
+  }
+
+  @objc
+  func tapCheckout() {
+    if items.count == 0 {
+      let ac = UIAlertController(title: "Cannot checkout", message: "Please add at least 1 item to the cart", preferredStyle: .alert)
+      ac.addAction(UIAlertAction(title: "Ok", style: .default))
+      self.present(ac, animated: true)
+    }
```

Comment:
This is not blocking, so the paysheet starts behind the error alert. add a `return` call after `self.present(..)` to prevent this